variables.tf

```
@@ -33,43 +33,12 @@ variable "enable_classiclink_dns_support" {
   default     = false
 }
 
-variable "security_group_enabled" {
+variable "enable_default_security_group_with_custom_rules" {
```

Wouldn't we use this naming convention instead ?

```suggestion
variable "default_security_group_with_custom_rules_enabled" {
```

---

@nitrocode The idea of this PR is to restore compatibility with version 0.25.0 of this module. This would be a breaking change, and I do not want any breaking changes at this point, because we will be introducing a set of breaking changes soon, including probably replacing this boolean altogether, and I want to minimize the number of times we require manual intervention, particularly in a module like this which is used by many of our other modules, which themselves may have to be modified to accommodate breaking changes.